docs/docs/reference/command-line-interface.md

```diff
@@ -13,6 +13,10 @@ For a better understanding of command line documentation syntax, the [docopt](ht
 
 The following options are available for all `meltano` subcommands:
```

Comment:
The `--cwd` is only available as a top-level CLI option, rather than for all `meltano` subcommands, so this isn't accurate. That said, the same applies to the `--log-level` and `--log-config` CLI options which are the only other things in this section, so maybe we just need to rename this from `Global Configuration` to `Top-Level Configuration`, and update the description text to read `The following CLI options are available for the top-level `meltano` command, rather than any particular sub-command.`